tidb-lightning/tidb-lightning-configuration.md

```diff
@@ -8,462 +8,677 @@ aliases: ['/docs/dev/tidb-lightning/tidb-lightning-configuration/','/docs/dev/re
+
+#### `pd-addr`
+
+- Specifies the address of any PD server from the cluster. Starting from v7.6.0, TiDB supports setting multiple PD addresses.
+
+<!-- Example: `"172.16.31.4:2379,56.78.90.12:3456"` -->
+
+#### `log-level`
+
+- Controls the log level of the TiDB library. TiDB Lightning imports TiDB as a library and generates some logs itself.
+
+<!-- Example: `"error"` -->
+
+#### `build-stats-concurrency`
+
+- Sets the TiDB session variable to speed up the Checksum and Analyze operations. For more information, see [Control `ANALYZE` concurrency](/statistics.md#control-analyze-concurrency).
+
+<!-- Example: `20` -->
+
+#### `distsql-scan-concurrency`
+
+- Sets the TiDB session variable to speed up the Checksum and Analyze operations. For more information, see [Control `ANALYZE` concurrency](/statistics.md#control-analyze-concurrency).
+- If [`checksum-via-sql`](#checksum-via-sql) is set to `"true"`, TiDB Lightning will execute the `ADMIN CHECKSUM TABLE <table>` SQL statement to perform the Checksum operation on TiDB. In this case, the following parameters `distsql-scan-concurrency` and `checksum-table-concurrency` will not take effect.
+
+<!-- Example: `15` -->
+
+#### `index-serial-scan-concurrency`
+
+- Sets the TiDB session variable to speed up the Checksum and Analyze operations. For more information, see [Control `ANALYZE` concurrency](/statistics.md#control-analyze-concurrency).
+
+<!-- Example: `20` -->
+
+#### `checksum-table-concurrency`
+
+- Sets the TiDB session variable to speed up the Checksum and `ANALYZE` operations. For more information, see [Control `ANALYZE` concurrency](/statistics.md#control-analyze-concurrency).
+- If [`checksum-via-sql`](#checksum-via-sql) is set to `"true"`, TiDB Lightning will execute the `ADMIN CHECKSUM TABLE <table>` SQL statement to perform the Checksum operation on TiDB. In this case, the following parameters `distsql-scan-concurrency` and `checksum-table-concurrency` will not take effect.
+
+<!-- Example: `2` -->
+
+#### `sql-mode`
+
+- Specifies the default SQL mode used to parse and execute the SQL statements.
+
+<!-- Example: `"ONLY_FULL_GROUP_BY,NO_AUTO_CREATE_USER"` -->
+
+#### `max-allowed-packet`
+
+- Sets maximum packet size allowed for SQL connections.
+- Set this to `0` to automatically fetch the `max_allowed_packet` variable from server on every connection.
+
+<!-- Example: `67_108_864` -->
+
+#### `tls`
+
+- Controls whether to use TLS for SQL connections.
+- Value options:
+    * `""`: force TLS (same as "cluster") if [`[tidb.security]`](#tidbsecurity) section is populated, otherwise same as `"false"`
+    * `"false"`: disable TLS
+    * `"cluster"`: force TLS and verify the server's certificate with the CA specified in the [`[tidb.security]`](#tidbsecurity) section
+    * `"skip-verify"`: force TLS but do not verify the server's certificate (insecure!)
+    * `"preferred"`: the same as `"skip-verify"`, but if the server does not support TLS, fall back to unencrypted connection
```

Comment:
```suggestion
    * `""`: forces TLS (the same as "cluster") if the [`[tidb.security]`](#tidbsecurity) section is populated. Otherwise, the same as `"false"`.
    * `"false"`: disables TLS.
    * `"cluster"`: forces TLS and verifies the server's certificate with the CA specified in the [`[tidb.security]`](#tidbsecurity) section.
    * `"skip-verify"`: forces TLS but does not verify the server's certificate. Note that this setting is insecure.
    * `"preferred"`: the same as `"skip-verify"`, but if the server does not support TLS, fall back to the unencrypted connection.
```